### ISSUE TITLE
VolumeDriver.Mount: exit status 1

### ISSUE THREAD

**Issue by PolyProgrammist** [lebokus/docker-volume-bindfs#4]
When I run:
```
docker volume create --driver lebokus/bindfs:latest -o sourcePath=$PWD -o map=$UID/1000:@$UID/@1000 some_volume > /dev/null
docker run -v some_volume:/home/ ubuntu:20.04     
```
I experience:                                                 
```                     
docker: Error response from daemon: VolumeDriver.Mount: exit status 1%!(EXTRA []interface {}=[]).
```

This may be connected with [this](https://github.com/sapk/docker-volume-gluster/issues/34) issue and [this](https://github.com/trajano/docker-volume-plugins/issues/29) issue 

(Reproduced on Mac OS Catalina 10.15.7, Docker version 20.10.10, latest version of bindfs plugin)

**Comment by StudioEtrange** [comment]
Yes I think it is the problem with the volume list not correctly managed.
This version should solved this https://github.com/StudioEtrange/docker-volume-bindfs
